HTShowcase/HTShowcase/ViewControllers/ShowcaseViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ShowcaseViewController.swift
+//  HTShowcase
+//
+//  Created by Gonzalo Barrios on 5/22/18.
+//  Copyright © 2018 Gonzalo Barrios. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+enum ViewLayout {
+    case common
+    case squared
+    case highImage
+}
+
+enum LayoutIconImageName: String {
+    case common = "ShowcaseView-common"
+    case squared = "ShowcaseView-squared"
+    case highImage = "ShowcaseView-highImage"
+}
+
+class ShowcaseViewController: UIViewController {
+    
+    //MARK: - IBOutlets
+    
+    @IBOutlet weak var showcaseCollectionView: UICollectionView!
+    
+    //MARK: - Vars
+    
+    var disposeBag = DisposeBag()
+    var viewModel: ShowcaseViewModel
+    var filterButton = UIBarButtonItem()
```

Comment:
You are overriding this, so it shouldn't be initalized empty here, it should just be declared